### PR TITLE
add HLTJetL1TMatchProducer as simplified stage2L1-version with corres…

### PR DESCRIPTION
**Pull request by kirschen**
…ponding filldescriptions and dropped Tau/Central/Forward inputs

### CODE REVIEW

**Comment by Martin-Grunewald** [comment]
We'd need a PR for 80X as well!

**Comment by Martin-Grunewald** [comment]
please test

**Comment by kirschen** [comment]
backport: #13469

File: HLTrigger/JetMET/src/HLTJetL1TMatchProducer.cc [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+  edm::Handle<l1t::JetBxCollection> l1Jets;
+  iEvent.getByToken(m_theL1JetToken,l1Jets);
+
+  typename TCollection::const_iterator jet_iter;
+  for (jet_iter = jets->begin(); jet_iter != jets->end(); ++jet_iter) {
+
+    bool isMatched=false;
+
+    for (unsigned int jetc=0;jetc<l1Jets->size();++jetc)
```
Comment (Martin-Grunewald) [inline HLTrigger/JetMET/src/HLTJetL1TMatchProducer.cc]: You have to select the l1t objects on BX=0 (or make BX configurable and code that part), see the example code in cms-l1t-offline#156

**Comment by Martin-Grunewald** [comment]
please test

**Comment by kirschen** [comment]
Selected the l1t objects on BX=0 as suggested by Martin. Here and in #13469

**Comment by Martin-Grunewald** [comment]
please test

**Comment by Martin-Grunewald** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**